controllers/front/ProductController.php

```diff
@@ -1228,6 +1228,18 @@ public function getTemplateVarPage()
         }
         $page['admin_notifications'] = array_merge($page['admin_notifications'], $this->adminNotifications);
 
+        $idProductAttribute = $this->getIdProductAttribute();
+        if ($idProductAttribute) {
+            $attributes = $this->product->getAttributeCombinationsById($this->getIdProductAttribute(), $this->context->language->id);
+            if ($attributes && count($attributes) > 0) {
+                $pageTitle = $page['meta']['title'];
+                foreach ($attributes as $attribute) {
+                    $pageTitle .= ' '.$attribute['group_name'].' '.$attribute['attribute_name'];
```

Comment:
Can directly concat to ` $page['meta']['title']`

Reply:
done!